en/test_and_ci/integration_testing.md

```diff
@@ -6,7 +6,12 @@ This topic explains how to run (and extend) PX4's ROS-based integration tests.
 [MAVSDK Integration Testing](../test_and_ci/integration_testing_mavsdk.md) is preferred when writing new tests.
 Use the ROS-based integration test framework for use cases that _require_ ROS (e.g. object avoidance).
 
-All PX4 integraton tests are executed automatically by our [Continuous Integration](../test_and_ci/continous_integration.md) system.
+All PX4 integration tests are executed automatically by our [Continuous Integration](../test_and_ci/continous_integration.md) system.
+:::
+
+:::note
+This page is about MAVROS integration tests.
+If you are looking for PX4 ROS 2 Interface Library integration tests, see [here](../ros2/px4_ros2_interface_lib.md#ci-integration-tests).
```

Comment:
This is a bit confusing, because I wouldn't expect people to be looking for this here.

Reply:
Agreed